### DISCUSSION TITLE
App Download

### DISCUSSION THREAD

**Discussion by NightXwl** [bridge-core/bridge.#149] [Q&A] [answered]
I wouldn’t say the problem is on your behalf, but more mine. 
I’m fairly new to github but I know bridge. will be the best tool to help me and therefore i’m here. However, i can’t seem to figure out how to install bridge. 

I’m on a mac, and i’m curious if anyone could take the time to make an explanation in steps to help me get it to my computer. English isn’t my main language, and that may be why I don’t really understand how to get it to work.

Problem is in short: I’m too stupid to understand how to get it installed.
And IF anyone has the time to explain how to get it to a mac that would be a pure lifesaver.

Thank you!

**Comment by solvedDev** [comment, marked as answer]
Hey 👋 
Please first choose a release from here: https://github.com/bridge-core/bridge./releases
Then, expand the assets section of the release and download the installer for your operating system

**Comment by NightXwl** [comment]
Med vänlig hälsning, My

**Comment by NightXwl** [comment]
Sorry about that first e-mail, its an automatic thing. Where do i find the installer

**Comment by solvedDev** [comment]
![image](https://user-images.githubusercontent.com/33347616/74836081-f3a4a780-531e-11ea-89cb-bb536b4c2cac.png)
![image](https://user-images.githubusercontent.com/33347616/74836161-1df66500-531f-11ea-86dc-cd9a75452bc3.png)
